stdeb/util.py

```diff
@@ -109,6 +109,9 @@ def check_call(*popenargs, **kwargs):
      'Allow installing into /some/random/virtualenv-path'),
     ('sign-results',None,
      'Use gpg to sign the resulting .dsc and .changes file'),
+    ('ignore-source-changes',None,
+     'Ignore all changes on source when building source package (add -i.* '
+     'option to dpkg-source'),
```

Comment:
Nitpick: please add the missing closing parenthesis in the help message.

Reply:
done